addons/warns.py

```diff
@@ -91,7 +119,8 @@ def __init__(self, bot):
         embed = discord.Embed(title="Warns for {}".format(target))
         count = 1
         for warn in warns:
-            embed.add_field(name="Warn #{}".format(count), value="{}".format(warn))
+            embed.add_field(name="Warn #{}".format(count), value="Date: {}\nReason: {}".format(time.strftime('%Y-%m-%d %H:%M:%S', time.localtime(warn['date'])),
```

Comment:
Better. One other change: do `**Date**: {}\n**Reason**:` instead, as they are hard to see. Also, drop the time component. That's not necessary for listwarns. Finally, switch date & reason. The reason in the more important part.

Reply:
imo listing the time a warn was given is useful for moderation purposes but I will switch them around.

Reply:
The exact time really won’t be. Just the date is. You can’t search by a specific time